payments/vipps/redirect.md

```diff
@@ -97,46 +83,253 @@ sequenceDiagram
+  activate Swedbank_Pay_FrontEnd
+  Swedbank_Pay_FrontEnd-->>Merchant: payment resource
+  deactivate Swedbank_Pay_FrontEnd
   Merchant-->>Browser: Redirect to payment page
-  note left of Browser:redirect to PayEx
-  Browser-->>PayEx_FrontEnd: enter mobile number
-  activate PayEx_FrontEnd
+  note left of Browser:redirect to Swedbank Pay
+  Browser-->>Swedbank_Pay_FrontEnd: enter mobile number
+  activate Swedbank_Pay_FrontEnd
 
-  PayEx_FrontEnd-->>Vipps_API: Initialize Vipps payment
+  Swedbank_Pay_FrontEnd-->>Vipps_API: Initialize Vipps payment
   activate Vipps_API
-  Vipps_API-->>PayEx_FrontEnd: response
-  PayEx_FrontEnd-->>Browser: Authorization response (State=Pending)
+  Vipps_API-->>Swedbank_Pay_FrontEnd: response
+  Swedbank_Pay_FrontEnd-->>Browser: Authorization response (State=Pending)
   note left of Browser: check your phone
   deactivate Merchant
 
   Vipps_API-->>Vipps_App: Confirm Payment UI
   Vipps_App-->>Vipps_App: Confirmation Dialogue
   Vipps_App-->>Vipps_API: Confirmation
-  Vipps_API-->>PayEx_BackEnd: make payment
-  activate PayEx_BackEnd
-  PayEx_BackEnd-->>PayEx_BackEnd: execute payment
-  PayEx_BackEnd-->>Vipps_API: response
-  deactivate PayEx_BackEnd
+  Vipps_API-->>Swedbank_Pay_BackEnd: make payment
+  activate Swedbank_Pay_BackEnd
+  Swedbank_Pay_BackEnd-->>Swedbank_Pay_BackEnd: execute payment
+  Swedbank_Pay_BackEnd-->>Vipps_API: response
+  deactivate Swedbank_Pay_BackEnd
   deactivate Vipps_API
-  PayEx_BackEnd-->>PayEx_FrontEnd: authorize result
-  PayEx_FrontEnd-->>Browser: authorize result
+  Swedbank_Pay_BackEnd-->>Swedbank_Pay_FrontEnd: authorize result
+  Swedbank_Pay_FrontEnd-->>Browser: authorize result
   Browser-->>Merchant: Redirect to merchant
   note left of Browser: Redirect to merchant
   activate Merchant
-  PayEx_FrontEnd-->>Merchant: Payment Callback
-  Merchant-->>PayEx_FrontEnd: GET <Vipps payments>
+  Swedbank_Pay_FrontEnd-->>Merchant: Payment Callback
+  Merchant-->>Swedbank_Pay_FrontEnd: GET <Vipps payments>
   note left of Merchant: Second API request
-  PayEx_FrontEnd-->>Merchant: Payment resource
-  deactivate PayEx_FrontEnd
+  Swedbank_Pay_FrontEnd-->>Merchant: Payment resource
+  deactivate Swedbank_Pay_FrontEnd
   Merchant-->>Browser: Display authorize result
   deactivate Merchant
 ```
 
+{:.code-header}
+**Request**
+
+```http
+POST /psp/vipps/payments HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "payment": {
+        "operation": "Purchase",
+        "intent": "Authorization",
+        "currency": "NOK",
+        "prices": [
+            {
+                "type": "Vipps",
+                "amount": 1500,
+                "vatAmount": 0
+            }
+        ],
+        "description": "Test Purchase",
+        "payerReference": "AB1234",
+        "userAgent": "Mozilla/5.0...",
+        "pageStripdown": true,
+        "language": "nb-NO",
+        "urls": {
+            "hostUrls": "http://test-dummy.net",
```

Comment:
I'm pretty sure `hostUrls` is an array. We should use `example.com` and `example.net` in our examples, not `test-dummy.net`.

```suggestion
            "hostUrls": ["https://example.com"],
```